std/digest/murmurhash.d

```diff
@@ -0,0 +1,860 @@
+/**
+Computes MurmurHash hashes of arbitrary data. MurmurHash is a non-cryptographic
```

Comment:
Link the first occurrence of the word MurmurHash to the appropriate Web reference.